src/database.ts

```diff
@@ -114,25 +114,24 @@ export const redshiftClusterMaintenanceSettings: ResourceValidationPolicy = {
             },
             preferredMaintenanceWindow: {
                 type: "string",
-                default: "Mon:09:30-Mon:10:00",
             },
             automatedSnapshotRetentionPeriod: {
                 type: "number",
-                default: 1,
             },
         },
     },
     validateResource: validateResourceOfType(aws.redshift.Cluster, (cluster, args, reportViolation) => {
-        const { allowVersionUpgrade, preferredMaintenanceWindow, automatedSnapshotRetentionPeriod } = args.getConfig<Required<RedshiftClusterMaintenanceSettingsArgs>>();
-        // Check the allowVersionUpgrade is configured properly TODO: below.
+        const { allowVersionUpgrade, preferredMaintenanceWindow, automatedSnapshotRetentionPeriod } = args.getConfig<RedshiftClusterMaintenanceSettingsArgs>();
```

Comment:
I removed the `Required` since not all the config properties have defaults. Ideally, inside the validation handler here, `allowVersionUpgrade` wouldn't be optional, since this one config property has a default value, it will always be set. But it's not that big of a deal in this case.